apps/monitoring/perftest/01/dynatrace-operator.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dynatrace-operator
+  namespace: monitoring
+spec:
+  releaseName: dynatrace-operator
+  interval: 5m
+  chart:
+    spec:
+      chart: dynatrace-operator
+      version: 0.4.2
+      sourceRef:
+        name: dynatrace-operator
+        kind: HelmRepository
+        namespace: monitoring
+  valuesFrom:
+    - kind: Secret
+      name: oneagent-values
```

Comment:
Same as base and should not be required.

Reply:
Keeping this code, in the event i test with an alternative namespace.

Reply:
In that case, the other elements apart from the namespace can be removed as these are already in the base.  Example

```
chart:
  spec:
    sourceRef:
      namespace: monitoring
```

Reply:
Reduced the code as suggested.